### DISCUSSION TITLE
Countdown Timer Reset

### DISCUSSION THREAD

**Discussion by shah-ut** [elementor/elementor#15464] [Feature Request]
Currently the practical implementation of a countdown timer is inefficient as the user needs to manually reset the timer every time it expires. 

An ideal feature would be to have the timer reset after a predetermined amount of time after expiry. Please revisit this and make this simple update. I have tried to code a custom workaround, but have come up short.

**Comment by benoitfouc** [comment]
You'r rigth, i'm rushing for this feature many times. It's like a loop countdown feature.
+1

**Comment by dvdhuidobro** [comment]
Here you go guys, a piece of code that deleted the countdown timer cookies when it gets to 0

```
<script>
jQuery(document).ready( function ($) {
	$(document).on( 'countdown_expire', function() {

      	Object.keys(localStorage)
        .filter(key => key.endsWith('evergreen_interval'))
        .forEach(key => localStorage
        .removeItem((key)))

      	Object.keys(localStorage)
        .filter(key => key.endsWith('evergreen_due_date'))
        .forEach(key => localStorage
        .removeItem((key)))

	});
});
</script>
```
